front/lib/api/assistant/global_agents.ts

```diff
@@ -101,7 +103,13 @@ async function getDataSourcesAndWorkspaceIdForGlobalAgents(
 }> {
   const owner = auth.getNonNullableWorkspace();
 
-  const defaultVaults = [await VaultResource.fetchWorkspaceGlobalVault(auth)];
+  const globalGroup = await GroupResource.fetchWorkspaceGlobalGroup(auth);
```

Comment:
Will refactor `fetchWorkspaceGlobalGroup` in a follow up PR to remove the Result type. In the meantime using an assert.